dist/rudder-sdk-js/README.md

```diff
@@ -104,133 +141,58 @@ rudderanalytics.track(
+If you are using a device mode destination like Heap, FullStory, etc., the JavaScript SDK needs to fetch the required configuration from the [**control plane**](https://docs.rudderstack.com/get-started/rudderstack-architecture#control-plane).
 
-To enable autotracking, make the load call as:
+If you are self-hosting the control plane using the [**RudderStack Control Plane Lite**](https://docs.rudderstack.com/get-started/control-plane-lite#what-is-the-control-plane-url) utility, your `load` call will look like the following:
 
-```jsx
-rudderanalytics.load(<YOUR_WRITE_KEY>, <DATA_PLANE_URL>, {
-  useAutoTracking: true,
+```javascript
+rudderanalytics.load(<WRITE_KEY>, <DATA_PLANE_URL>, {
+  configUrl: <CONTROL_PLANE_URL>,
 });
 ```
 
-By default, the SDK doesn't capture any DOM element values. To start capturing values, like form field values when submitting the form, other input element values etc, whitelist them using any attribute of the element for which you want to send values, For ex, tracking element values for all elements whose any one attribute is "CLASS_ATTR_NAME":
+> More information on how to get the `CONTROL_PLANE_URL` can be found [**here**](https://docs.rudderstack.com/get-started/control-plane-lite#what-is-the-control-plane-url).
 
-```jsx
-rudderanalytics.load(<YOUR_WRITE_KEY>, <DATA_PLANE_URL>, {
-  useAutoTracking: true,
-  valTrackingList: ["CLASS_ATTR_NAME"],
-});
-```
-
-# [](https://github.com/rudderlabs/rudder-sdk-js/blob/master/README.md#self-hosted-config-plane)Self-hosted Config Plane
-
-If you are using a device mode destination like Heap, FullStory, etc., the JavaScript SDK needs to fetch the required configuration from the Control Plane. If you are using the RudderStack Config Generator to host your own Control Plane, then follow [this guide](https://docs.rudderstack.com/how-to-guides/rudderstack-config-generator) and make a `load` call as shown:
-
-```jsx
-rudderanalytics.load(<YOUR_WRITE_KEY>, <DATA_PLANE_URL>, {configUrl: CONTROL_PLANE_URL});
-```
+| **For detailed technical documentation and troubleshooting guide on the RudderStack’s JavaScript SDK, check out our [docs](https://docs.rudderstack.com/stream-sources/rudderstack-sdk-integration-guides/rudderstack-javascript-sdk).** |
+| :------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
 
-# [](https://github.com/rudderlabs/rudder-sdk-js/blob/master/README.md#contribute)Contribute
+## Adding your own integrations
 
-You can start adding integrations of your choice for sending data through their JavaScript SDKs.
+You can start adding integrations of your choice for sending the data through their respective web (JavaScript) SDKs.
 
-## [](https://github.com/rudderlabs/rudder-sdk-js/blob/master/README.md#how-to-build-the-sdk)How To Build the SDK
+### How to build the SDK
```

Comment:
This section is applicable only for repository readme file whereas this is bundled with the distribution package. Please remove it.